### PR TITLE
Minor English tweaks to quick-start.md

### PR DESCRIPTION
**Pull request by msabramo**
# Description

Minor mods to English wording in `quick-start.md`

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by msabramo** [comment]
@microsoft-github-policy-service agree

File: docs/how-to-guides/quick-start.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Quick start
 
-This guide will walk you through the fist step using of prompt flow code-first experience.
+This guide will walk you through the first steps of the prompt flow code-first experience.
 
 **Prerequisite** - To make the most of this tutorial, you'll need:
-- Know how to program with Python :)
+- Python programming knowledge
```
Comment (wangchao1230) [inline docs/how-to-guides/quick-start.md:6]: could we merge this change into PR 3461? so we have 1 instead of 2 PR for the same purpose

Comment (msabramo) [reply docs/how-to-guides/quick-start.md:6]: I’m okay with that but I’m on vacation so it will be a while before I’m at a keyboard and can do that.

Comment (msabramo) [reply docs/how-to-guides/quick-start.md:6]: Oh looks like the other one was merged already

Comment (wangchao1230) [reply docs/how-to-guides/quick-start.md:6]: Yes. The original one is merged already. I think we can just merge this one also.

**Pull request merged**